class UserThrowTest
{
    def method() {
        throw Exception("AAA");
    }
    
    def method2() {
        method();
    }
}
